src/app/services/push.service.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { TestBed, inject } from '@angular/core/testing';
+import { HttpClientModule, HttpClient } from '@angular/common/http';
+import { PushService } from './push.service';
+
+describe('PushService', () => {
```

Comment:
Tests technically do not test anything significant.

How do you know that proper calls are made and functions are called accordingly? Please try to cover more cases.

Reply:
I tried to use already existing mock data inside ```shared/mocks/feedItem.mock.ts```, but there comes some issue with some of the parameters inside individual: ```search_metadata```, ```statuses```, and ```aggregations```. I tried to make it work out, but it is taking longer time than expected. Currently the simple test checks that push service is being created. I will add tests in later patches (Because many other parts need new tests since the project is upgraded, and the tests written earlier can't be taken as it is, I have to write new ones). I want to fix sidebar, newsTab issue firstly before this weekend. Hope that works?